### ISSUE TITLE
GMS dependency

### ISSUE THREAD

**Issue by marado** [austrianredcross/stopp-corona-android#137]
#41 refers to the implementation of GAEN (Google/Apple's API). Not only that change would make the app dependent on the non-free GMS (with all the limitations to public scrutiny that come along with it), it would also be a blocker to things like #3 .

**Comment by mitsuhiko** [comment]
Unfortunately there is really no way around that. In theory one could reimplement the system frameworks on Android and given how many countries are using that technology now I would assume someone will try to reimplement it.

**Comment by ironjan** [comment]
FYI: there's a project trying to replace/loosen this dependency.

> EDIT: tl;dr: We are working an an open implementation of the Google/Apple protocol (I call it PPCP, which may be wrong). You can find and contribute to our repository here: https://github.com/theScrabi/CoraLibre-android-sdk

-- https://github.com/corona-warn-app/cwa-app-android/issues/75

**Comment by theScrabi** [comment]
Yes. We try to reimplement GAEN (or PPCP as I called it)  with the same API as GMS, therefore our solution might not only be compatible to german corona warn app, but also to stopp corona app. However we are not including our solution into the official app, we fork it instead. Our fork will be compatible to other apps, and will be able to talk to the official servers. I don't know if forking would be possible for stopp corona app as well.

If you want to take a look, help is wellcome: https://coralibre.de

**Comment by firestorm99** [comment]
or it may be easier porting their integration of parts of microg to app. 

https://codeberg.org/corona-contact-tracing-germany/cwa-android

It's a pity that the Austrian app is not capable of interacting with the apps of other countries yet.
